bigquery_etl/shredder/config.py

```diff
@@ -0,0 +1,239 @@
+    client_id_target(table="telemetry_derived.events_live"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.experiments_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.main_events_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.main_summary_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.block_autoplay_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.crash_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.downgrade_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.event_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.first_shutdown_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.focus_event_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.frecency_update_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.health_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.heartbeat_v4"): CLIENT_ID_SRC,
+    client_id_target(
+        table="telemetry_stable.main_v4",
+        cluster_conditions=tuple(
+            (condition, needs_clustering)
+            for condition, needs_clustering in chain(
+                {
+                    f"sample_id = {sample_id} AND normalized_channel = 'release'": False
+                    for sample_id in range(100)
+                }.items(),
+                [("normalized_channel != 'release'", True)],
+            )
+        ),
+    ): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.modules_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.new_profile_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.saved_session_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.shield_icq_v1_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.shield_study_addon_v3"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.shield_study_error_v3"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.shield_study_v3"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.testpilot_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.third_party_modules_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.untrusted_modules_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.update_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.voice_v4"): CLIENT_ID_SRC,
+    # activity stream
+    impression_id_target(
+        table="activity_stream_stable.impression_stats_v1"
+    ): IMPRESSION_ID_SRC,
+    impression_id_target(
+        table="activity_stream_stable.spoc_fills_v1"
+    ): IMPRESSION_ID_SRC,
+    # TODO check if this table's impression_id is from activity stream
+    impression_id_target(
+        table="messaging_system_stable.undesired_events_v1"
+    ): IMPRESSION_ID_SRC,
+}
+
+UNSUPPORTED = [
```

Comment:
Why are these unsupported? Is this a temporary list while details get sorted out?